hooks/command

```diff
@@ -43,14 +46,19 @@ read_caches_from() {
 
         for region in "${regions[@]}"; do
 
-          echo "Logging in to ECR, region: ${region}"
-          ecr_login "$region" "$account_id"
+          if ! { [ "${CURRENT_LOGGED_IN_REGION}" = "${region}" ] && [ "${CURRENT_LOGGED_IN_ACCOUNT}" = "${account_id}" ]; }; then
+            echo "Logging in to ECR, region: ${region}"
+            ecr_login "$region" "$account_id"
+            CURRENT_LOGGED_IN_REGION="${region}"
+            CURRENT_LOGGED_IN_ACCOUNT="${account_id}"
+          fi
 
           cache="$(get_ecr_url "${image}" "${account_id}" "${region}"):${tag}"
 
-          docker pull "${cache}" || true
-          caches_from+=('--cache-from' "${cache}")
-
+          if docker pull "${cache}"; then
+            caches_from+=('--cache-from' "${cache}")
+            break
```

Comment:
Did you test it? Did it work? Last time I tried this, the cache didn't work as the tag was different. Event though it is same image `...us-west-2....` and `ap-southeast-2...` is not same tag

Reply:
You would probably need to tag the image will all regions.

Reply:
I don't understand. Let's say tag is `master-latest`. We try to cache from all regions. After discussing with @JonCubed we've concluded that we assume that image was uploaded by very same plugin (this one), and it is exactly same image (with same digest). Scenarios where these images are different are too weird and unlikely for us to consider

Reply:
We don't have "regional" tags, nor we need to